src/lib/stepper/stepper.md

```diff
@@ -156,6 +156,38 @@ by placing a `matStepperIcon` for each of the icons that you want to override. T
 
 Note that you aren't limited to using the `mat-icon` component when providing custom icons.
 
+#### Step States
+You can set the state of a step to whatever you want. The given state by default maps to an icon. However, it can be overridden the same way as metioned above.
+
+```html
+<mat-horizontal-stepper>
```

Comment:
we should make this an embedded example

Reply:
Are you referring to adding it in the `demo-app`?

Reply:
you can put it under the `material-examples` directory and then indicate where it should be embedded in the markdown file using a comment like this: `<!-- example(example-name) -->`. That way when we generate the docs site page the example will get inserted. You can preview the example in the demo app at http://localhost:4200/examples